internal/db/schema/migrations/postgres/10/01_sentinels.up.sql

```diff
@@ -0,0 +1,33 @@
+begin;
+
+  create function wt_is_sentinel(string text)
+    returns bool
+  as $$
+    select length(trim(leading u&'\fffe ' from string)) > 1 AND starts_with(string, u&'\fffe');
+  $$ language sql
+     immutable
+     returns null on null input;
+  comment on function wt_is_sentinel is
+    'wt_is_sentinel returns true if string is a sentinel value';
+
+  create domain wt_sentinel as text
+    constraint wt_sentinel_not_valid
+      check(
+        wt_is_sentinel(value)
+        or
+        length(trim(u&'\fffe ' from value)) > 1
```

Comment:
why the space after `\fffe` here and in other places in this file?

The comment says "non-empty" so why > 1 instead of > 0 ?

Reply:
Copy and paste error. Ty!